tools/build/scanCode.cfg

```diff
@@ -0,0 +1,54 @@
+# scanCode.py configuration file
+
+# List of filenames containing the text of valid license (headers)
+# These files SHOULD be in the same directory path where scanCode.py
+# resides.
+[Licenses]
+ASFLicenseHeader.txt
+ApacheIBMLicenseHeader.txt
+
+# Filters (path/filename) with wildcards and associated scan checks
+# that are to be run against them.  The checks are actual valid
+# function names found in scanCode.py.
+[Includes]
+*=is_not_symlink
+*.scala=has_block_license, no_tabs, no_trailing_spaces, eol_at_eof
+*.py=no_tabs, no_trailing_spaces, eol_at_eof
+*.java=has_block_license, no_tabs, no_trailing_spaces, eol_at_eof
+*.js=no_tabs, no_trailing_spaces, eol_at_eof
+*.gradle=no_tabs, no_trailing_spaces, eol_at_eof
+*.md=no_tabs, eol_at_eof
+*.go=has_block_license, no_tabs, no_trailing_spaces, eol_at_eof
+build.xml=no_tabs, no_trailing_spaces, eol_at_eof
+deploy.xml=no_tabs, no_trailing_spaces, eol_at_eof
+
+# List of paths (inclusive of subdirectories) to exlude from code scanning
+[Excludes]
+# General exclusions
+.tox
+.git
+# OpenWhisk exclusions
+bin
+tests/build/reports
+tests/dat
+# openwhisk-catalog exclusions (Python samples)
+packages/samples/hello
+# The following repos. have so far been identified as having scanning errors
+# and will be excluded until their owners have completed updates to add Travis
```

Comment:
I like the idea of having exclusion file in another repo  
But exclusion should be for build artifacts and such not main source code. 
The catalog already have Travis and runs scan code 

For this particular case it's only the hello sample let's fix the hello sample should be super easy and remove this exclusion before we merge this. Let's take the time for this take there is no rush 

And again Matt please know that I really really thank you on your contributions on this making OpenWhisk great!!